src/main/java/org/elasticsearch/action/bulk/BulkRequest.java

```diff
@@ -493,6 +495,25 @@ public void writeTo(StreamOutput out) throws IOException {
             request.writeTo(out);
         }
         out.writeBoolean(refresh);
+        out.writeBoolean(durable);
         timeout.writeTo(out);
     }
+
+    /**
+     * Returns <code>true</code> iff this request ensures that all changes written to the transaction log and
+     * are synced to the underlying storage on all receiving shards once the request returns. Defaults to <code>true</code>
+     */
+    public boolean isDurable() {
+        return durable;
+    }
+
+    /**
+     * Set to <code>true</code> iff this request should sync all changes written to the transaction log
+     * to the underlying storage on all receiving shards once the request returns. If this is set to <code>false</code> data
+     * durability is not guaranteed for the time window the transaction log is synced asynchronously.
+     * Defaults to <code>true</code>.
+     */
+    public void setDurable(boolean durable) {
+        this.durable = durable;
+    }
```

Comment:
you mean you will fix every getter and setter in the java api? I didnt know we finally made a decision around this... would love to have a single way to do things throughout the codebase though